site/website/blog/2020-10-06-enabling-full-text-search.md

```diff
@@ -0,0 +1,492 @@
+---
+title: Enabling Full-text Search with Change Data Capture in a Legacy Application
+description: Debezium based Change Data Capture sources for Hazelcast Jet
+author: František Hartman
+authorURL: https://www.linkedin.com/in/frantisek-hartman/
+authorImageURL: https://i.stack.imgur.com/3X7wE.png 
+---
+
+In this post, we will take a legacy application and improve its search
+functionality. We will do this with only a few changes to the
+application itself - the technique called change data capture  (CDC)
```

Comment:
rephrase this a bit, somehow this dashed section doesn't flow together with the rest, breaks the flow of thought